### ISSUE TITLE
Healthy-normalized cross-entropy fitness function (for Logistic Regression)

### ISSUE THREAD

**Issue by SebastianRehfeldt** [lukasjf/mphp#30]
http://scikit-learn.org/stable/modules/feature_selection.html

**Comment by SebastianRehfeldt** [comment]
Example - Logistic Regression:
Use healthy data as objective when training classifiers

Another example:
Use filter to reduce possible features
apply wrapper method
